src/insight/insightClient.ts

```diff
@@ -266,6 +267,13 @@ export class CoveoInsightClient {
         });
     }
 
+    public logCreateArticle(createArticle: CreateArticleMetadata, metadata?: CaseMetadata) {
+        return this.logCustomEvent(
+            InsightEvents.createArticle,
+            metadata ? {...generateMetadataToSend(metadata, false), ...createArticle} : {createArticle}
```

Comment:
`createArticleMetadata` is already an object, we would get a different structure in the event custom data if we keep it `{createArticle}`
```suggestion
            metadata ? {...generateMetadataToSend(metadata, false), ...createArticle} : createArticle
```